portal/templates/profile_macros.html

```diff
@@ -462,6 +462,69 @@ <h4 class="text-success registration-label">{{_('registered')}}</h4>
      });</script>
     {%- endif -%}
 {%- endmacro %}
+{% macro profileEmailLog(person) -%}
+    <div id="emailLogContainer">
+        <div id="emailLogContent"></div>
+        <div id="emailLogMessage" class="error-message"></div>
+    </div>
+    <script>
```

Comment:
Could this be moved into admin.js too? :)

Reply:
@ivan-c  Ivan, I moved the JS code to main.js - I think it can be re-used by other pages if need to. Let me know if you see any issue with the change.

Reply:
@ivan-c Ivan, I realized that I needed to add email content sent as well for this UI.  I will let u know when I am done here.

Reply:
Cool, thanks for giving it a try. No worries if it's a headache to rework.

Reply:
@ivan-c Ivan, this PR is ready for reviewing again.  Let me know if you find any issue.